### PR TITLE
python.pkgs.pytest-check: init at 0.3.5

### PR DESCRIPTION
**Pull request by flokli**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by B4dM4n** [comment]
LGTM.

Builds fine as `python2Packagespytest-check` and `python3Packages.pytest-check` on NixOS and macOS.

File: pkgs/development/python-modules/pytest-check/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    owner = "okken";
+    repo = "pytest-check";
+    rev = version;
+    sha256 = "11wb4f4sp4cr5mzqdakrbycwgfr2p1sx1l91fa6525wnfvgc0qy3";
+  };
+
+  checkInputs = [ pytest ];
+
+  propagatedBuildInputs = [
+    pytest
```
Comment (worldofpeace) [inline pkgs/development/python-modules/pytest-check/default.nix]: We're preferring having `pytest` in `buildInputs` to make using multiple versions of pytest more convenient.

Comment (worldofpeace) [reply pkgs/development/python-modules/pytest-check/default.nix]: Can we also use `pytestCheckHook` in `checkInputs` to eliminate overriding the `checkPhase`?

Comment (flokli) [reply pkgs/development/python-modules/pytest-check/default.nix]: Updated the PR. Didn't know about `pytestCheckHook`, thanks!

**Comment by worldofpeace** [comment]
Thanks @flokli :sparkles:

**Pull request merged**